static_src/components/app_quicklook.jsx

```diff
@@ -0,0 +1,112 @@
+
+import React from 'react';
+
+import EntityIcon from './entity_icon.jsx';
+import PanelRow from './panel_row.jsx';
+import { appStates } from '../constants.js';
+import createStyler from '../util/create_styler';
+import style from 'cloudgov-style/css/cloudgov-style.css';
+
+const EXTRA_INFO = [
+  'state',
+  'memory',
+  'diskQuota'
+];
+
+const propTypes = {
+  app: React.PropTypes.object.isRequired,
+  orgGuid: React.PropTypes.string.isRequired,
+  spaceGuid: React.PropTypes.string.isRequired,
+  spaceName: React.PropTypes.string,
+  extraInfo: React.PropTypes.arrayOf((propVal) => EXTRA_INFO.includes(propVal))
+};
+
+const defaultProps = {
+  spaceName: '',
```

Comment:
I feel like setting empty string as the default for optional props doesn't add much value. React is pretty good at handling `undefined` in that it basically renders as empty string. I would prefer to leave this prop out of the defaults.